sonar-php-plugin/src/test/java/org/sonar/plugins/php/PHPSensorTest.java

```diff
@@ -193,7 +222,7 @@ private void analyseFileWithException(PHPCheck check, InputFile inputFile, Strin
     thrown.expect(AnalysisException.class);
     thrown.expectMessage(expectedMessageSubstring);
     try {
-      sensor.analyseFiles(phpAnalyzer, ImmutableList.of(inputFile), progressReport);
+      sensor.analyseFiles(context, phpAnalyzer, ImmutableList.of(inputFile), progressReport);
     } finally {
       verify(progressReport).cancel();
     }
```

Comment:
@ivandalbosco we could also test `describe` method, see e.g. https://github.com/SonarSource/sonar-javascript/blob/master/sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/JavaScriptSquidSensorTest.java#L113